### ISSUE TITLE
Opening URL from command line if Min is already open doesn't work.

### ISSUE THREAD

**Issue by OppositeNote** [minbrowser/min#773]
When I try to open an URL in Min from the command line (or QuiteRSS), Min doesn't open the page if it's already running. It does gain focus. No errors are shown in both the command line from which Min is originally run or the one in which Min is run with the URL argument.
It does open the link in a new tab when Min is not running though.

OS: Arch Linux
Min version: According to my package manager Min is version 1.10.1-1, according to Min itself it's version 1.10.0.

**Comment by PalmerAL** [comment]
I can reproduce this; it looks like the issue is that in `handleCommandLineArguments` here: https://github.com/minbrowser/min/blob/master/main/main.js#L61, we assume that the URL is always the second argument, but when you launch a second instance in arch, the arguments look like this:
```
['/usr/bin/electron', '/usr/bin/min', 'http://example.com']
```
So we probably need to search all the arguments for something that looks like a URL.

As an aside, this is the first time I've ever actually installed the arch package, and it seems to be fairly messed up. First of all, it seems to have been built from commit aae39f98b787e36a9fa5dbc56b13312619a07732 instead of 493c43faad6278bb57119a6abc89c8659d38519d which is causing the displayed version number to be wrong, even though it is actually 1.10.1. Secondly, 10.10 is supposed to run on Electron 5.0, but the package is depending on Electron 4. This means you're running an older version of Chromium than you should be (69 instead of 73), and because Electron 5 included API changes that have been adopted in 1.10, features like reader mode and full-text search are completely broken in the arch package.

For now, I would recommend building Min from source instead of using the package, but I'm hoping this can be fixed; do you have any idea who I should contact about this?

**Comment by dbfreem** [comment]
I found this link. It has some more information about the package. 

https://www.archlinux.org/packages/community/any/min/

**Comment by OppositeNote** [comment]
Thanks for the reply, I'll try building from source and let you know how it goes.

The reason it depends on Electron4 is probably because [the version of Electron in the repo](https://www.archlinux.org/packages/community/x86_64/electron/) (which is flagged out of date) is Electron 4.  I have found [this todo](https://www.archlinux.org/todo/electron-5/), which I think means Electron 5 will be in the community repo shortly ([It's in community-staging already](https://www.archlinux.org/packages/community-staging/x86_64/electron/))? The todo dates back a month or two so I expect it should be there in the near future. I actually have no idea how packages get transferred from community-staging to community though.

**Comment by dbfreem** [comment]
Is this still an issue?

@PalmerAL  should the readme somehow say something about these packages in the different package managers that are not official packages?

**Comment by PalmerAL** [comment]
I believe it's still an issue, I'll fix it shortly.

Adding a note about the packages might work, but I'm not sure everyone would read it. Right now, I'm thinking that we should check the electron version on startup, and if the major version doesn't match `electronVersion` in package.json, we should crash and output an error. That would hopefully ensure that packages are built correctly, and since Min will most likely be broken if it's run with the wrong version of Electron, I doubt it would be much of a functional loss anyway.

**Comment by PalmerAL** [comment]
Fixed in bad8b00932728fee1aafa7a43e2454817dff305f.
